modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextBar.vue

```diff
@@ -40,17 +40,19 @@ export default {
   name: 'TheAposContextBar',
   mixins: [ AposPublishMixin, AposAdvisoryLockMixin ],
   data() {
-    // Since cookie-based login sessions and sessionStorage are not precisely the same
-    // thing, correct any forbidden combination at page load time
-    if (window.apos.mode === 'published') {
-      window.sessionStorage.setItem('aposEditMode', JSON.stringify(false));
+    const query = apos.http.parseQuery(location.search);
+    // If the URL references a draft, go into draft mode but then clean up the URL
+    const draftMode = query['apos-mode'] || 'published';
```

Comment:
the `draftMode` var can be `'published'`? should this variable just be `mode` ?

Reply:
or, isn't this really the version we're referencing? should it be `version` ?

Reply:
This is a previously established and released convention in the component, so I'd like to not reopen the subject of the name of this variable, at least in this ticket.